fuzzers/test_utils.py

```diff
@@ -131,10 +131,10 @@ def test_initialize_env_in_var_with_sanitizer(fs):
         '-fsanitize=address -fsanitize=array-bounds,bool,builtin,enum,'
         'float-divide-by-zero,function,integer-divide-by-zero,null,object-size,'
         'return,returns-nonnull-attribute,shift,signed-integer-overflow,'
-        'unsigned-integer-overflow,unreachable,vla-bound,vptr -O3')
+        'unreachable,vla-bound,vptr -O3')
     assert env.get('CXXFLAGS') == (
         '-fsanitize=address -fsanitize=array-bounds,bool,builtin,enum,'
         'float-divide-by-zero,function,integer-divide-by-zero,null,object-size,'
         'return,returns-nonnull-attribute,shift,signed-integer-overflow,'
-        'unsigned-integer-overflow,unreachable,vla-bound,vptr -stdlib=libc++ '
+        'unreachable,vla-bound,vptr -stdlib=libc++ '
         '-O3')
```

Comment:
I wonder if `-O3` is why some bugs we find on OSS-Fuzz don't repro here. We don't use O3 in OSS-Fuzz.

Reply:
Should i switch to -O1 ?

Reply:
Done, for bug based benchmarking, switched to -O1.

Reply:
Not sure, probably the right call.
Thanks!